Nodejs/Product/Nodejs/Debugger/DebugEngine/Remote/NodeRemoteEnumDebugProcesses.cs

```diff
@@ -103,18 +103,24 @@ public NodeRemoteEnumDebugProcesses(NodeRemoteEnumDebugProcesses processes)
                         break;
                     }
                 } catch (OperationCanceledException) {
-                    LiveLogger.WriteLine("NodeRemoteEnumDebugProcesses ping timed out.");
+                    LiveLogger.WriteLine("OperationCanceledException connecting to remote debugger");
                 } catch (DebuggerAlreadyAttachedException ex) {
+                    LiveLogger.WriteLine("DebuggerAlreadyAttachedException connecting to remote debugger");
                     exception = ex;
                 } catch (AggregateException ex) {
+                    LiveLogger.WriteLine("AggregateException connecting to remote debugger");
                     exception = ex;
                 } catch (IOException ex) {
+                    LiveLogger.WriteLine("IOException connecting to remote debugger");
                     exception = ex;
                 } catch (SocketException ex) {
+                    LiveLogger.WriteLine("SocketException connecting to remote debugger");
                     exception = ex;
                 } catch (WebSocketException ex) {
+                    LiveLogger.WriteLine("WebSocketException connecting to remote debugger");
                     exception = ex;
                 } catch (PlatformNotSupportedException) {
+                    LiveLogger.WriteLine("NodeRemoteEnumDebugProcesses ping timed out.");
```

Comment:
PlatformNotSupportedException doesn't have anything to do with the ping timing out, does it?